README.adoc

```diff
@@ -1,34 +1,125 @@
-////
-This is the base template for Hazelcast integration module guides.
-
-You can clone this repository, work on it and create your guide, and then push to a new repository.
-////
-
-:github-address: [github-repo-address-of-your-guide]
+:github-address: https://github.com/hazelcast-guides/getting-started-hazelcast-springboot
 :templates-url: https://raw.githubusercontent.com/hazelcast-guides/adoc-templates/master
+:hazelcast: Hazelcast IMDG
+:framework: Spring Boot
 
-= Title
+= Getting Started with Hazelcast using Spring Boot
 
-// Content entered directly below the header but before the first section heading is called the preamble.
+This guide will help you to start using Hazelcast by building a Spring Boot application.
+
+include::{templates-url}/what-is-hz.adoc[]
 
 include::{templates-url}/link-to-repo.adoc[]
 
 == What You’ll Learn
 
-// Define the problem and tell reader what he/she will learn from this guide.
+include::{templates-url}/microservices/what-you-will-learn.adoc[]
 
 == Prerequisites
 
-// Define the prerequisites for the guide.
+- ~15 minutes
+include::{templates-url}/microservices/prerequisites.adoc[]
+
+== Using Hazelcast in the Spring Boot Application
+
+include::{templates-url}/microservices/application-structure.adoc[]
```

Comment:
Shouldn't this part be moved to "What you'll learn"? Or Maybe create a section "What you will build"?

![image](https://user-images.githubusercontent.com/2834997/78643801-2138b680-78b5-11ea-9620-3ce4857379f8.png)

Reply:
I don't think it will suit for "What you'll learn". We can create a separate section for it, but it is not that much detailed info.

Reply:
The thing that looks weird to me is that I read "Using Hazelcast in the Spring Boot Application", but you actually start explaining what you'll build in this sample. So, I'd either add a section here or move it somewhere.

Reply:
Moved it to a new section 👍